### ISSUE TITLE
Beregningsfejl

### ISSUE THREAD

**Issue by HBnyvej** [rndfm/elspotpris#148]
Jeg har checket elprisen for d. 17/3-2023 kl 9:00 på jeres Elspotpris-side.
Jeg anvender OK El Flex, Radius og DK2. 
Jeg får den til 1,29 mens Elspotpris siger 1,15.
De følgende anvendte priser er alle exclusive moms, hvor notationen er mit beløb / jeres beløb.
Anvendte beløb: Spotpris 0.244/0.24, Elafgift 0.008/0.01, Transmission 0.112/0.112, OK Flex 0.1057/0.1057, Radius 0.566/0.566.
Summen bliver excl moms 1.0357/1.0337, og incl moms 1.294625/1.292125.
Jeg går ud fra, at der skal betales moms af samtlige priser.

**Comment by rndfm** [comment]
Bemærk at priserne på siden og i udregningseksemplet skifter når man vælger moms til og fra.

Du har momsen på to gange for radius transport da 0,566 er inkl moms.
Elafgift står til 1 øre inkl moms eller 0,8 ex moms på elspotpris.dk

Jeg fandt dog en anden fejl da jeg dobbelttjekkede siden. Det ser ikke ud til at prisen for spottillæget skifter visning(med/uden moms) i regneeksemplet på https://elspotpris.dk/live - Den regner dog rigtigt

**Comment by HBnyvej** [comment]
Ja, du har ret på Radius hjemmeside kan de aktuelle tariffer ses med/uden moms, mens samtlige historiske tariffer er excl moms.
Nu stemmer mine beregninger på øre.
Beklager støjen.
